### ISSUE TITLE
merge this original repo with the monome fork

### ISSUE THREAD

**Issue by montoyamoraga** [dinchak/grid-studies-nodejs#4]
hi @dinchak, i was checking out the monome fork of this repo and it has more recent commits than this one.

my question is, would you consider making the monome repo the parent one of this one? or even merging them altogether for no redundancy? 

also, i want to add pull requests with credits to the people that have worked on the different grid studies on the READMEs of the repo, so i will issue those when they are ready :)

**Comment by dinchak** [comment]
i'm on board with whatever is best for people! agreed something should be done to reduce the confusion.  i'm happy with taking the easiest path there -- should i just delete this repo?

**Comment by montoyamoraga** [comment]
@dinchak yeah i think i would have just the repo on monome, since it's what most people will find on the internet, and it keeps track of all and everyone's contributions, and we can all collaborate and maintain it there :)

**Comment by dinchak** [comment]
ok great i will delete it! thanks!
